src/_update.sh

```diff
@@ -23,6 +23,7 @@ function mod_update() {
   pull_docker_images
   mod_start
   mod_check
+  checkFailedContainers
```

Comment:
Example of grouping the logic here:

```
maxTries=2
for attempt in {1..$maxTries}; do
  mod_start

  unhealthyContainers=$(checkFailedContainers 2>&1)
  if [[ $? -eq 0 ]]; then
    debug "No failed containers found, continuing"
    break # exit the loop if all is kosher
  fi
  debug "${unhealthyContainers}"

  if [[ $attempt >= $maxTries ]]; then
    error "Max attempts failed, some containers are still unhealthy"
    exit 1

  else;
    info "An error occured with one or more containers, attempting to start again"
    sleep 5
  fi
mod_check
```

Reply:
of course, you can just sub in the `compose_client ps` here and no extra function.